lib/engine/game/g_1812/game.rb

```diff
@@ -1,23 +1,23 @@
 # frozen_string_literal: true
 
+require_relative '../g_1867/game'
 require_relative 'entities'
 require_relative 'map'
 require_relative 'meta'
-require_relative 'phases'
-require_relative 'trains'
 require_relative '../base'
 require_relative '../company_price_up_to_face'
+require_relative '../../loan'
+require_relative '../interest_on_loans'
 
 module Engine
   module Game
     module G1812
-      class Game < Game::Base
+      class Game < G1867::Game
         include_meta(G1812::Meta)
-        include Entities
-        include Map
-        include Phases
-        include Trains
+        include G1812::Entities
+        include G1812::Map
```

Comment:
These don't need to be qualified, do they?
```suggestion
        include Entities
        include Map
```